### PR TITLE
Add items: clean workspaces and clear descriptions

### PR DESCRIPTION
**Pull request by nickmeinhold**
This was an attempt to capture what we spoke about today.  The purpose is to be a starting point for discussion rather than as a suggestion of what I think the end product should look like.  

There is no rush on this, please feel free to take the time to think about it if you would like to.  

Thanks!

### CODE REVIEW

File: .contributing/code-of-conduct.md [hunk 1]
```diff
@@ -34,6 +40,17 @@ TRY to be a Clean Team Member
 
 PREFER using Asana over email 
 
+DO keep workspaces highly organised 
+- Asana workspaces are public spaces, meaning the state we leave the space in has an effect on the rest of the team 
+- if a workspace/section/column has a purpose, remove anything that doesn't serve that purpose 
+- we each have a column in the CrowdLeague Workflow workspace, with the purpose of describing our day - what we're currently working on and the plan for what comes after that 
+
+DO attempt to communicate effectively when creating tasks/issues/etc. 
+- so that other team members (and you) can read the tasks/issues/etc. quickly and have a good idea of what it is about
```
Comment (tamari-gray) [inline .contributing/code-of-conduct.md:49]: love it

File: .contributing/code-of-conduct.md [hunk 2]
```diff
@@ -34,6 +40,17 @@ TRY to be a Clean Team Member
...
+DO keep workspaces highly organised 
+- Asana workspaces are public spaces, meaning the state we leave the space in has an effect on the rest of the team 
+- if a workspace/section/column has a purpose, remove anything that doesn't serve that purpose 
+- we each have a column in the CrowdLeague Workflow workspace, with the purpose of describing our day - what we're currently working on and the plan for what comes after that 
+
+DO attempt to communicate effectively when creating tasks/issues/etc. 
+- so that other team members (and you) can read the tasks/issues/etc. quickly and have a good idea of what it is about  
+- it can be helpful to: 
+  - break down large tasks/issues into smaller pieces 
+  - try and read what you've written with the mindset of someone who doesn't have all the context that you currently do
```
Comment (tamari-gray) [inline .contributing/code-of-conduct.md:52]: nice point

**Review by tamari-gray — APPROVED** [review]
lgtm!!!

**Review by T-Davis — APPROVED** [review]
Ah, great work @nickmeinhold! Thanks for continually keeping us moving in the right direction.
GuTeam GTD!

**Comment by nickmeinhold** [comment]
Woohoo!  Thanks heaps guys!  🙂

**Pull request merged**